healthCheck/gotenberg.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * name : gotenberg.js.
+ * author : Deepa.
+ * created-date : 04-Feb-2021.
+ * Description : Gotenberg health check.
+*/
+
+// Dependencies 
+const request = require('request');
+
+function health_check() {
+    return new Promise(async (resolve, reject) => {
+
+        let options = {
+            headers: {
+              "Content-Type": "application/x-www-form-urlencoded"
+            }
+        }
+
+        request.get(process.env.GOTENBERG_URL, options, gotenbergCallback);
```

Comment:
@deepakr123 - Why not use "/ping" which is health check endpoint for gotenberg

Reply:
Refer - https://github.com/thecodingmachine/gotenberg/blob/master/docs/openapi.yaml